doc/guides/west/moving-to-west.rst

```diff
@@ -0,0 +1,88 @@
+.. _moving-to-west:
+
+Moving to West
+##############
+
+With Existing Zephyr Repository
+*******************************
+
+To convert a "pre-west" Zephyr setup on your computer to west, follow these
+steps.
+
+#. Install west.
+
+   On Linux::
+
+     pip3 install --user -U west
+
+   On Windows and macOS::
+
+     pip3 install -U west
+
+   For details, see :ref:`west-install`.
+
+#. Move your zephyr repository to a new :file:`zephyrproject` parent directory,
+   and change directory there.
+
+   On Linux and macOS::
+
+     mkdir zephyrproject
+     mv zephyr zephyrproject
+     cd zephyrproject
+
+   On Windows ``cmd.exe``::
+
+     mkdir zephyrproject
+     move zephyr zephyrproject
+     chdir zephyrproject
+
+   The name :file:`zephyrproject` is arbitrary. You can choose any name,
+   as long as there are no spaces anywhere in the path.
+
+#. Create a :ref:`west installation <west-installation>` using the zephyr
+   repository as a local manifest repository::
+
+     west init -l zephyr
+     west update
+
+Troubleshooting
```

Comment:
I don't think this section belongs here.

I have 2 suggestions:

1. Let's create a global FAQ. I think that is the better approach
2. Let's create a west Troubleshooting section that is a separate page